.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
mozda bi mogao i neki tabsize = 4 da se doda

Reply:
Bas sam to hteo da pitam danas :)